### PR TITLE
chore: adjust fetching contributors script

### PR DESCRIPTION
**Pull request by mainframev**
It's just started to add automatically "actions-users" after we added rebase, so I excluded that 
 Storybook: https://orbit-mainframev-chore-change-contributors.surge.sh

### CODE REVIEW

File: docs/plugins/contributors/fetchedUsers.json [hunk 1]
```diff
@@ -239,21 +239,6 @@
     "position": "",
     "github": "https://github.com/infi-pc"
   },
-  {
-    "id": 2528846,
```
Comment (DSil) [inline docs/plugins/contributors/fetchedUsers.json:243]: Why is this contributor being removed? Is it expected?

Comment (mainframev) [reply docs/plugins/contributors/fetchedUsers.json:243]: yep, not manually, the script removed him 🤷🏻

File: docs/package.json [hunk 2]
```diff
@@ -124,6 +123,7 @@
     "use-resize-observer": "^8.0.0",
     "uuid": "^8.3.2",
     "vfile-reporter": "^7.0.5",
-    "webpack": "^5.76.0"
+    "webpack": "^5.76.0",
+    "gatsby-plugin-postcss": "^6.11.0"
```
[the remark this answers is not in the document]
Comment (mainframev) [reply docs/package.json:127]: I will rerun it to add it in proper order :D 
Let's not be so strict about what should be a part of PR if it relates to such small thing

[the remark this answers is not in the document]
Comment (DSil) [reply docs/package.json]: > I just moved it from dev deps to normal ones

That is not marked as a change, hence my question 🤔 I saw a dependency being added that is unrelated to the PR and I questioned it.
After further investigation, I see that on master we don't have it. We only have it on the Tailwind base branch, that's why it's not marked as a change here, only an addition.

**Review by DSil — APPROVED** [review]

File: docs/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (mainframev) [reply docs/package.json]: Aha,  right! Thanks

**Pull request merged**